app/controllers/BtaStubController.scala

```diff
@@ -22,16 +22,23 @@ import config.AppConfig
 import play.api.i18n.{I18nSupport, MessagesApi}
 import play.api.mvc.{Action, AnyContent}
 import services.BtaStubService
+import uk.gov.hmrc.auth.core.{AuthConnector, AuthorisedFunctions, NoActiveSession}
 import uk.gov.hmrc.play.bootstrap.controller.FrontendController
 
+import scala.concurrent.Future
+
 @Singleton
-class BtaStubController @Inject()(val messagesApi: MessagesApi,
-                                  btaStubService: BtaStubService,
-                                  implicit val appConfig: AppConfig) extends FrontendController with I18nSupport {
+class BtaStubController @Inject()(val messagesApi: MessagesApi, val authFunctions: AuthorisedFunctions,
+                                  val authConnector: AuthConnector, btaStubService: BtaStubService, implicit val appConfig: AppConfig)
+  extends FrontendController with I18nSupport {
 
-  val landingPage: Action[AnyContent] = Action.async { implicit request =>
-    btaStubService.getPartial().map { partial =>
-      Ok(views.html.btaStub.landingPage(partial))
+  def landingPage(): Action[AnyContent] = Action.async { implicit request =>
+    authFunctions.authorised() {
+      btaStubService.getPartial().map { partial =>
+        Ok(views.html.btaStub.landingPage(partial))
+      }
+    }.recoverWith {
+      case _: NoActiveSession => Future.successful(Unauthorized)
```

Comment:
If this returns `Unauthorized` it means it just shows a blank page which means we can't acceptance test hitting our 'dummy' BTA stubbed page and potentially should redirect to somewhere else - can discuss in person if that is needed at all - if not this is fine